.github/workflows/flutter-build.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: '12.x'
-      - run: git clone https://github.com/flutter/flutter.git --depth 1 -b v1.12.13+hotfix.5 _flutter
+      - run: git clone https://github.com/flutter/flutter.git --depth 1 -b v1.12.13+hotfix.6 _flutter
       - run: echo "::add-path::$GITHUB_WORKSPACE/_flutter/bin"
```

Comment:
Agreed that we should change from beta (hotfix 5) to stable (hotfix 6). I'm wondering if we should go to hotfix 9 as this includes some other fixes. Though Flutter has a policy of very limited scope of hotfixes, I still have consternation as the latest hotfix is from April 1st and only a few days old: https://github.com/flutter/flutter/wiki/Hotfixes-to-the-Stable-Channel#hotfix9-april-01-2020

Reply:
Let me run some tests first.  We had an issue with very recent stable builds breaking us before.

Reply:
Alright we can move up to hotfix 8 with the build working fine.

Think we should give hotfix 9 a bit of time to bake before we upgrade to it.